src/index.css

```diff
@@ -2,5 +2,10 @@
 @import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css");
 
 body {
-  background-color: var(--white-light);
+  background-color: #162538;
+  color: white;
```

Comment:
You could use variables here

Reply:
The color is used just once in the code so I used it directly